lib/Chat/Notifier.php

```diff
@@ -314,8 +314,16 @@ public function removePendingNotificationsForRoom(Room $chat, bool $chatOnly = f
 		// @todo this should be in the Notifications\Hooks
 		$notification->setApp('spreed');
 
-		$notification->setObject('chat', $chat->getToken());
-		$this->notificationManager->markProcessed($notification);
+		$objectTypes = [
```

Comment:
This function is called when the chat history is cleared, not sure we want to remove recording notifications and information at that case?